src/opts/cli.rs

```diff
@@ -43,32 +47,36 @@ pub fn command(scale_help: String, default_theme: Option<ThemeType>) -> Command
         .value_hint(ValueHint::AnyPath)
         .help("Path to the markdown file");
 
-    let mut theme_arg = Arg::new("theme")
+    let theme_arg = Arg::new("theme")
         .short('t')
         .long("theme")
         .number_of_values(1)
         .value_parser(value_parser!(ThemeType))
         .help("Theme to use when rendering");
-    if let Some(theme) = default_theme {
-        theme_arg = theme_arg.default_value(theme.as_str());
-    }
 
     let scale_arg = Arg::new("scale")
         .short('s')
         .long("scale")
         .number_of_values(1)
         .value_parser(value_parser!(f32))
-        .help(scale_help);
+        .help(SCALE_HELP);
 
     let gen_comp_arg = Arg::new("shell")
         .long("gen-completions")
         .help("Generate shell completions")
         .number_of_values(1)
         .value_parser(value_parser!(Shell));
 
-    let page_width_arg = Arg::new("page_width")
+    let config_arg = Arg::new("config")
+        .short('c')
+        .long("config")
+        .number_of_values(1)
+        .value_parser(value_parser!(PathBuf))
+        .help("Configuration file to use");
+
+    let page_width_arg = Arg::new("page width")
```

Comment:
```suggestion
    let page_width_arg = Arg::new("page_width")
```

The arg reads a little awkward in the help text with the space in the center (`-w, --page-width <page width>`)